tests/TKSetupSamples.m

```diff
@@ -0,0 +1,36 @@
+//
+//  TKSetupSamples.m
+//  TokenSdkTests
+//
+//  Created by Larry Hosken on 10/25/17.
+//  Copyright © 2017 Token Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "TKTestBase.h"
+
+#import "TokenIOSync.h"
+#import "TokenIOBuilder.h"
+
+#import "TKUtil.h"
+
+@interface TKSetupSamples : TKTestBase
+
+@end
+
+@implementation TKSetupSamples
+
+- (void)testCreateSDKClient {
+    [self run: ^(TokenIOSync *tIO) {
```

Comment:
you might not need this block. this block is useless if you don't need the tIO object.

Reply:
Excellent, done.

Reply:
If I just remove the block, the call to createMember never returns. Looking at a comment in TKTestBase, I guess it's because "gRPC posts to the main dispatch queue, so we have to run it for the callbacks to be completed." When I look at some things on the internet, I think if you're writing an app, the app does something so that things don't get stuck on the queue.

Maybe there's a better way than TKTestBase run to get this working... but I'm new and don't know what it is :-D